lib/helper.js

```diff
@@ -565,7 +565,7 @@ helper = {
 
     // map RAML schema security schema and PM request auths. set null by default.
     auth.type = _.get(authTypeMap, _.get(securitySchemes[securedBy], 'type'), null);
-    securitySchemes[securedBy].description && (auth.describe(securitySchemes[securedBy].description));
+    _.has(securitySchemes[securedBy], 'description') && (auth.describe(securitySchemes[securedBy].description));
```

Comment:
Any chance of `securitySchemes` to not be an object? We could have typeerror in such case as well.

We can use `typeof _.get(securitySchemes, '[${securedBy}].description)' === 'string'` for more safer access.

Reply:
In this case, `securitySchemes` was an empty string, but we don't get a type error when we resolve `[securedBy]`. But this does make sense, I'll see if I can modify it to be safer.